src/main/java/org/cactoos/scalar/Mapped.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.scalar;
+
+import java.io.IOException;
+import org.cactoos.Func;
+import org.cactoos.Scalar;
+
+/**
+ * {@link Scalar} that apply a {@link Func} to the result
+ * of another {@link Scalar}.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * <p>This class implements {@link Scalar}, which throws a checked
+ * {@link IOException}. This may not be convenient in many cases. To make
+ * it more convenient and get rid of the checked exception you can
+ * use the {@link Unchecked} decorator.</p>
+ *
+ * @param <T> Type of wrapped scalar result
+ * @param <U> Type of result
+ * @since 1.0.0
+ */
+public final class Mapped<T, U> implements Scalar<U> {
```

Comment:
@victornoel could you add tests along with the new class?

Reply:
@victornoel seen the change, thank you. I think that it would be better to add new commits when possible instead of force-pushing to make the work of `REV` and `QA` easier.

Reply:
@scristalli interesting, I hold the exact opposite opinion :) I think a clean git historic is better for the project. Also you can always see the changes between the various revision of the PR via the link on the words "force-pushed" that github gives you, like this you can properly do your job (and be exactly sure of the diff actually, I could have added one more commit but modified one of the previous one without you noticing ;) and we get a cleaner git commit history.
![image](https://user-images.githubusercontent.com/160975/61173410-51a42c00-a593-11e9-92d6-2f403d82e8bf.png)

Reply:
@victornoel we could agree to disagree on the Git matter, but it gets worse when we move on to GitHub.

1. The "force-pushed" link that you pointed at is buggy: although it works fine here, for [cactoos-matchers#131](https://github.com/llorllale/cactoos-matchers/pull/131) it lists some very weird changes for me, probably because it compares with one of the branches on my own fork on the repository. Not anyone's fault here, but as a `REV` I would like to read code and not worry about additional complications.

2. Additionally, as I commented in [cactoos-matchers#131](https://github.com/llorllale/cactoos-matchers/pull/131), force-pushing might have made one `REV` comment just disappear, which would actually be a major issue (although still due to a GitHub bug (feature?)). Admitted that I have not made a mistake, and even if there is some weird way to get it back for `REV` and `QA`, I think that it definitely makes both jobs a lot harder to say the least.